formatters/gocov/gocov.go

```diff
@@ -70,14 +69,15 @@ func (r Formatter) Format() (formatters.Report, error) {
 			}
 			blocks[lstIdx].Count += b.Count
 		}
+		lineNum := 1
```

Comment:
can't start lines be 0 ? 🤔

Reply:
They used to, but we changed them to be 1-based for better readability (lines in an editor start from 1 usually). This way `for lineNum < b.StartLine` reads like "do this and that for all lines before start of block". If lines are zero-based the statement becomes `for lineNum < b.StartLine - 1` which at least for me is harder to read. But I can change it to start from zero if you prefer it that way :).

Reply:
oh, I see. Thanks for the explanation. However, there might be folks out there using the `test-reportoer` downloading  it's latest version. If they haven't upgraded the `gocov` versio they are using things will start to behave differently for them once we merge this. Could we make it to be `for lineNum < b.StartLine - 1` for this reason ? I mean, for backwards compatibility.

Reply:
It is actually backwards compatible this way. Iterating from `0` to startLine - `1` is effectively the same as iterating from `1` to `startLine`. The index itself is not used anywhere, but is just a counter, so it cannot affect anything user facing.

Reply:
![facepalm-really](https://user-images.githubusercontent.com/11605222/144914883-90e14420-12b8-49dd-bc33-442b908513c4.gif)